packages/babel-cli/src/babel/index.js

```diff
@@ -243,7 +243,7 @@ delete opts.outFile;
 delete opts.outDir;
 delete opts.copyFiles;
 delete opts.includeDotfiles;
-delete opts.quiet;
+delete opts.verbose;
```

Comment:
was this change requested somewhere? whats the motivation behind it?

Reply:
discussed here: https://github.com/babel/babel/issues/6303#issuecomment-369064626

Reply:
👍